src/main/java/step2/domain/Judge.java

```diff
@@ -0,0 +1,47 @@
+package step2.domain;
+
+import step2.dto.JudgeResponse;
+import step2.dto.Lotto;
+import step2.dto.Price;
+import step2.dto.ShopResponse;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Judge {
+
+    private static final int MATCH_CASE = 6 + 1;
```

Comment:
```suggestion
    private static final int MATCH_CASE = 7;
```

굳이 6 + 1 로 하신 이유가 있나요? 만약 가독성 때문이라면, 변수명을 통해서 그 의미를 나타내는 게 더 좋다고 생각합니다.